docs/learn-faq.md

```diff
@@ -70,6 +70,21 @@ The Web3 Foundation will distribute up to 20% of mainnet DOTs prior to network l
 
 Mainnet DOT tokens are not transferrable until mainnet launch, expected in early 2020. Any transfers before that time of mainnet DOTs are illegitimate and unauthorized. DOTs can not be moved from a current allocation address. Individuals with an allocation of DOTs who transfer their DOT address to someone else can always keep a copy of their private key, therefore there is extreme risk for individuals participating in transfers of DOTs before mainnet launch.
 
+### What are the ways to find out the minimum stake necessary for the validators?
+
+There are few ways to estimate it.
+
+1. [Offline Phragmen](https://github.com/kianenigma/offline-phragmen)
```

Comment:
```suggestion
- [Offline Phragmen](https://github.com/kianenigma/offline-phragmen)
```

Reply:
Numbered list won't work if the paragraph explaining each list item isn't also indented under the list. As it is, markdown would make each number into 1. rather than 1, 2, 3.